pelita/scripts/pelita_main.py

```diff
@@ -249,7 +260,7 @@ def long_help(s):
                            help='Stop when a bot has been killed.')
 
 viewer_settings = parser.add_argument_group('Viewer settings')
-viewer_settings.add_argument('--geometry', type=geometry_string, metavar='NxM',
+viewer_settings.add_argument('--geometry', type=w_h_string, metavar='NxM',
```

Comment:
Edge case: This allows `pelita --geometry small`

Reply:
yes... I think it's ok to live with this? You can't always prevent people from shooting themselves in the foot...